js/tests/unit/util/index.spec.js

```diff
@@ -324,22 +324,6 @@ describe('Util', () => {
       expect(Util.isVisible(div)).toEqual(false)
     })
 
-    it('should return true if an ancestor element is visibility hidden, but reverted', () => {
```

Comment:
Based on this https://codepen.io/GeosSV/pen/abJmGGw, your test should work.

Reply:
Correct, that test would have passed based on how the computed visibility works. Would you like me to add back this test?

Reply:
why not. it was nice :smiley: